R/bayesplot-helpers.R

```diff
@@ -469,3 +469,53 @@ grid_lines_y <- function(color = "gray50", size = 0.2) {
 overlay_function <- function(...) {
   stat_function(..., inherit.aes = FALSE)
 }
+
+
+
+# Resolve a function name and store the expression passed in by the user
+#' @noRd
+#' @param f a function-like thing: a string naming a function, a function
+#'   object, an anonymous function object, a formula-based lambda, and `NULL`.
+#' @param fallback character string providing a fallback function name
+#' @return the function named in `f` with an added `"tagged_expr"` attribute
+#' containing the expression to represent the function name and an
+#' `"is_anonymous_function"` attribute to flag if the expression is a call to
+#' `function()`.
+as_tagged_function <- function(f = NULL, fallback = "func") {
```

Comment:
This function works as described, just one question: how come sometimes you call rlang functions with `rlang::foo()` and other times just `foo()`? I think either way works since we have `@import rlang` in `bayesplot-package.R`, just curious if your choices were intentional.

Reply:
Oops. Probably an artifact of me using typing `rlang::[TAB]` to use autocomplete to find the right rlang function.